js/index.js

```diff
@@ -1,7 +1,71 @@
 // Iteration 1: Names and Input
+ const hacker1 = "America"
+console.log("The drivers name is " + hacker1)
+
+const hacker2 = "Carlos"
+console.log("The navigators name is " + hacker2)
+
 
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length){
+  console.log("The driver has the longest name, it has " + hacker1.length + " characters." )
+} else if (hacker2.length > hacker1.length) {
+console.log("It seems that the navigator has the longest name, it has " + hacker2.length + " characters")
+} else {
+  console.log("Wow, you both have equally long names, " + hacker1.length + " characters")
+} 
+
+// Iteration 3: Loops
+let driver = ''
+for(let i=0; i < hacker1.length; i++){
```

Comment:
Aqui puedes agregar el uppercase dentro de la funcion  

`for(let i=0; i < hacker1.length; i++){
  driver += hacker1[i].toUpperCase() + ' '
}`